app/lib/activitypub/tag_manager.rb

```diff
@@ -66,7 +66,9 @@ def to(status)
       [COLLECTIONS[:public]]
     when 'unlisted', 'private'
       [account_followers_url(status.account)]
-    when 'direct', 'limited'
+    when 'limited'
+      []
```

Comment:
I felt that it might be better to consistently put "people textually tagged in the post" into `cc` instead of `to`, and indeed, leaving `to` to something like the group actor you mention.